CHANGES.rst

```diff
@@ -713,6 +713,8 @@ csv_tools
 cube_build
 ----------
 
+- Added dq flagging [#3804]
```

Comment:
Is this truly the first and only DQ flagging that's been in `cube_build`? Or is it just the addition of special flagging for holes and non-slice regions? If the latter, then perhaps the log info should be more specific.

Reply:
This is the first time the DQ flag has been sent in cube_building.